temp/static_analysis.md

```diff
@@ -0,0 +1,549 @@
+```pycon
+>>> print(selection.spatial_selection)
+TBD
+```
+
+### Intersect two spatial selections
+```pycon
+>>> space_sel_1 = selection_1.spatial_selection
+>>> print(space_sel_1)
+Spatial selection with 3 node(s):
+[1, 2, 3]
+>>> space_sel_2 = selection_2.spatial_selection
+>>> print(space_sel_2)
+Spatial selection with 3 node(s):
+[3, 4, 5]
+>>> space_sel_1.intersect(space_sel_2)
+>>> print(space_sel_1)
+Spatial selection with 1 node(s):
+[3]
+```
+
+### Union of two spatial selections
+```pycon
+>>> selection_3 = Selection()
+>>> selection_3.select_nodes(nodes=[1, 2, 3])
+>>> selection_4 = Selection()
+>>> selection_4.select_nodes(nodes=[3, 4, 5])
+>>> space_sel_3 = selection_3.spatial_selection
+>>> space_sel_4 = selection_4.spatial_selection
+>>> space_sel_3.union(space_sel_4)
+>>> print(space_sel_3)
+Spatial selection with 5 node(s):
+[1, 2, 3, 4, 5]
+```
+Do we then need to explicitly update the spatial_selection of selection_1?
+
+### Select based on constructed geometry
+```pycon
+>>> from ansys.dpf.core.geometry import Line
+>>> line = Line(coordinates=[[1.0, 0.0, 1.0], [5.0, 0.0, 1.0]])
+>>> selection_5 = Selection()
+>>> selection_5.select_geometry(line)
+>>> print(selection_5)
+Selection with:
+- 0 steps
+
+- 100 nodes 
+[...]
+```
+Right now a Line element has an underlying mesh with nodes and elements. We could print out this information.
+
+### Set a selection as default 
+for a simulation for further result extractions
+```pycon
+>>> static_simulation.activate_selection(selection=selection)
+>>> print(static_simulation.active_selection)
+Selection with:
+- 0 steps
+
+- 0 node/elements
+ 
+```
+
+### Remove a selection as default 
+for a simulation for further result extractions
+```pycon
+>>> static_simulation.deactivate_selection()
+>>> print(static_simulation.active_selection)
+None 
+```
+
+### Apply a SpatialSelection to a Simulation 
+to get resulting list of entities IDs (nodes or elements)
+```pycon
+>>> list_of_IDs = selection.spatial_selection.apply_to(static_simulation)
+>>> print(list_of_IDs)
+[ ... ]
+```
+
+## Mesh operations
+
+### Extract the mesh
+```pycon
+>>> mesh = static_simulation.mesh
+>>> print(mesh)
+DPF Mesh:
+    XXXX nodes
+    XXXX elements
+    Unit: m
+    with solid (3D) elements
+```
+
+### Plot the mesh 
+with defaults for static:
+- not deformed
+- at last step
+```pycon
+>>> mesh.plot(
+...     opacity=0.3, title="Static mesh plot", text="defaults to not deformed, last step"
+... )
+<Image>
+```
+
+### Plot elements orientations with triads
+```pycon
+>>> mesh.plot(triads=True)
+<Image>
+```
+
+### Map a load onto the mesh
+A load is a DataObject, so has a mesh support. Thus:
+```pycon
+>>> load_2 = mesh.interpolate(load)
+```
+
+## Extract specific results
+
+Arguments:
+- components: list of components amongst
+  - [1, 2, 3, 4, 5, 6] equivalent to ["X", "Y", "Z", "XY', "YZ", "XZ"], or "N" for the norm
+  - [1, 2, 3] for S1, S2, S3, E1, E2, or E3 when requesting principal stresses/strains
```

Comment:
@germa89 
I am not sure I understand which 1, 2, 3 you refer to when you say it would be confusing?

Also, the way I am designing it now, "all" would be equivalent to not specifying components.
As for "norm", I am not sure how to treat it.
It could be indeed:
- an additional component
- an additional parameter, which means we take the norm of the components asked?